{
    "alarms": [
        {
            "index": 5500,
            "cause": "SOFTWARE_ERROR",
            "name": "ASTAIRE_PROCESS_FAIL",
            "levels": [
                {
                    "severity": "CLEARED",
                    "details": "The Astaire process has been restored to normal operation.",
                    "description": "Astaire: Process failure cleared.",
                    "cause": "The Astaire process has been restored to normal operation. The previously issued alarm has been cleared.",
                    "effect": "Memcached synchronization for cluster nodes is fully operational.",
                    "action": "No action."
                },
                {
                    "severity": "CRITICAL",
                    "details": "Monit has detected that the Astaire process has failed. A restart will automatically be attempted. If this alarm does not clear, the Astaire process may have been stopped or an unrecoverable failure may have occurred.",
                    "description": "Astaire: Process failure",
                    "cause": "Monit has detected that the Astaire process has failed. A restart will automatically be attempted.",
                    "effect": "Synchronization of the Memcached database will be delayed until the process recovers.",
                    "action": "If this alarm does not clear, the Astaire process may have been stopped or an unrecoverable failure may have occurred. If the alarm fails to clear then contact your support representative."
                }
            ]
        },
        {
            "index": 5501,
            "cause": "DATABASE_INCONSISTENCY",
            "name": "ASTAIRE_RESYNC_IN_PROGRESS",
            "levels": [
                {
                    "severity": "CLEARED",
<<<<<<< HEAD
                    "details": "The local Memcached resynchronization process has ended, no longer blocking a scale-up or down operation (a different type of resynchronization operation may still be in progress though). An error log will indicate if the process was unsuccessful.",
                    "description": "Astaire: Memcached resynchronization complete",
                    "cause": "The local Memcached process is synchronized with the rest of the cluster. Memcached resynchronization is no longer blocking a scale-up or scale-down operation (a different type of resynchronization operation may still be in progress though.",
                    "effect": "The local node has received all of the synchronized data.",
                    "action": "No action."
                },
                {
                    "severity": "MINOR",
                    "details": "Astaire is resynchronizing the local Memcached process with the rest of the cluster. Service should be unaffected. If a scale-up or scale-down operation is in progress, wait for resynchronization to finish before completing the operation.",
                    "description": "Astaire: Memcached database resynchronization in progress",
                    "cause": "Astaire is resynchronizing the local Memcached process with the rest of the cluster.",
                    "effect": "Service is unaffected.",
                    "action": "If a scale-up or scale-down operation is in progress, wait for resynchronization to finish before completing the operation."
                }
            ]
        },
        {
            "index": 5502,
            "cause": "UNDERLYING_RESOURCE_UNAVAILABLE",
            "name": "ASTAIRE_MEMCACHED_COMM_ERROR",
            "levels": [
                {
                    "severity": "CLEARED",
                    "details": "Astaire communication to at least one Memcached has been restored.",
                    "description": "Astaire: Memcached communication error cleared"
                },
                {
                    "severity": "CRITICAL",
                    "details": "Astaire has failed to make a connection with all Memcached processes in the cluster. If this alarm does not clear, network connection may have been lost, or the Memcached processes may have failed and be in an unrecoverable condition.",
                    "description": "Astaire: Memcached communication error"
                }
            ]
        },
        {
            "index": 5503,
            "cause": "UNDERLYING_RESOURCE_UNAVAILABLE",
            "name": "ASTAIRE_VBUCKET_ERROR",
            "levels": [
                {
                    "severity": "CLEARED",
                    "details": "Astaire communication to Memcached for a piece of data has been restored.",
                    "description": "Astaire: Memcached vBucket communication error cleared"
                },
                {
                    "severity": "MAJOR",
                    "details": "Astaire is unable to contact any Memcacheds for a piece of data. Some subscriber data will be unavailable. If this alarm does not clear, ensure that all Memcacheds are operational and that network connectivity exists between them and Astaire.",
                    "description": "Astaire: Memcached vBucket communication error"
                }
            ]
        }
    ]
}
